calc_from_args_utils/calculation_methods.py

```diff
@@ -0,0 +1,531 @@
+#!/usr/bin/env python
+import logging
+from argparse import Namespace
+
+import numpy as np
+import pandas as pd
+import tabulate
+from genologics.entities import Process
+
+from calc_from_args_utils.udf_arg_methods import (
+    fetch_from_arg,
+    get_UDF_source,
+    get_UDF_source_name,
+)
+from epp_utils import formula, udf_tools
+
+DESC = """This file contains the method functions for a UDF-agnostic script."""
+
+
+def volume_to_use(process: Process, args: Namespace):
```

Comment:
I think these methods should document in the docstring which attributes of `args` it will use and for what. So that one can figure out how to use it?

Reply:
ef5483a28ac56d95a47603ed7a77341ec0c1a312